sismic/code/python.py

```diff
@@ -43,17 +43,19 @@ def __iter__(self):
         return iter(self.__frozencontext)
 
 
-def create_send_function(event_list: List[Event]) -> Callable[..., None]:
+def create_send_function(event_list: List[Event], event_class: Event) -> Callable[..., None]:
```

Comment:
I suggest to use ``event_klass`` instead of ``event_class`` for consistency with, e.g., ``evaluator_klass`` used in ``Interpreter``. 
I also think that the type of ``event_class`` should be something like ``Callable[(str, ...), Event]`` or simply ``Callable[..., None]`` for consistency with the return type of ``create_send_function``.

Finally, L57 should be removed as the "right" ``event`` is created on L58, right?